serializers/src/glTF/2.0/Extensions/KHR_lights_punctual.ts

```diff
@@ -0,0 +1,185 @@
+                }
+                if (this._exporter._glTF.extensionsRequired.indexOf(NAME) == -1) {
+                    this._exporter._glTF.extensionsRequired.push(NAME);
+                }
+            }
+            if (this._exporter._glTF.extensions == null) {
+                this._exporter._glTF.extensions = {};
+            }
+            this._exporter._glTF.extensions[NAME] = this._lights;
+        }
+    }
+    public postExportNodeAsync(context: string, node: INode, babylonNode: Node): Nullable<Promise<INode>> {
+        return new Promise((resolve, reject) => {
+            if (babylonNode instanceof ShadowLight) {
+                let babylonLight: ShadowLight = babylonNode;
+                let light: ILight;
+
+                let lightType = (
+                    babylonLight.getTypeID() == Light.LIGHTTYPEID_POINTLIGHT ? LightType.POINT : (
+                        babylonLight.getTypeID() == Light.LIGHTTYPEID_DIRECTIONALLIGHT ? LightType.DIRECTIONAL : (
+                            babylonLight.getTypeID() == Light.LIGHTTYPEID_SPOTLIGHT ? LightType.SPOT : null
+                        )));
+                if (lightType == null) {
+                    Tools.Warn(`${context}: Light ${babylonLight.name} is not supported in {NAME}`);
+                }
+                else {
+                    let lightPosition = babylonLight.position.clone();
+                    if (!lightPosition.equalsToFloats(1.0, 1.0, 1.0)) {
+                        if (this._exporter._convertToRightHandedSystem) {
+                            lightPosition.z *= -1;
+                        }
+                        node.translation = lightPosition.asArray();
+                    }
+                    if (babylonLight.falloffType != Light.FALLOFF_GLTF) {
+                        Tools.Warn(`${context}: Light falloff for ${babylonLight.name} does not match the ${NAME} specification!`);
+                    }
+                    light = {
+                        type: lightType
+                    };
+                    if (!babylonLight.diffuse.equalsFloats(1.0, 1.0, 1.0)) {
+                        light.color = babylonLight.diffuse.asArray();
+                    }
+                    if (!(babylonLight.intensity == 1.0)) {
+                        light.intensity = babylonLight.intensity;
+                    }
+                    if (!(babylonLight.range == Number.MAX_VALUE)) {
+                        light.range = babylonLight.range;
+                    }
+
+                    if (lightType === LightType.SPOT) {
+                        let babylonSpotLight = babylonLight as SpotLight;
+                        if (!(babylonSpotLight.angle == Math.PI / 2.0)) {
+                            if (light.spot == null) {
+                                light.spot = {};
+                            }
+                            light.spot.outerConeAngle = babylonSpotLight.angle / 4;

```

Comment:
This doesn't seem right. It should be the opposite of the loader, no?

```suggestion
                            light.spot.outerConeAngle = babylonSpotLight.angle / 2;
```

Reply:
@bghgary yeah should be / 2